### DISCUSSION TITLE
Ghalper lite

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#3117] [Q&A]
Hello Serge, I switched to an HP model device and installed ghalper. The graphics card panel is fully functional, so I am able to undervolt my GPU.

If it doesn't create extra work for you, could you turn it into an application that focuses only on the GPU part? Honestly, your program seems like the best GPU limiting and application solution so far.

Additionally, I found a similar application for my device called omenmon. It's available on GitHub. I wish it were as functional and good as your software. Hopefully, you will work on HP models in the future.

**Comment by seerge** [comment]
@hadixcanimm there is no need for a separate version of the app. As you have discovered by yourself current version of the app can run on non-asus devices as well. 

Of course all proprietary features of asus laptops won't be available. But you can use GPU section for overclocking / clock limiting already :)
